targets/TARGET_NUVOTON/TARGET_M480/mbed_lib.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
shall this be part of target  configuration ? (in targets.json) file? it would be all in one place. we already have there config for some targets.